ci-cd/github-actions.md

```diff
@@ -247,18 +247,18 @@ Now let's change the Docker Hub login with the GitHub Container Registry one:
 ```
 {% endraw %}
 
-Remember to change how the image is tagged. The following example keeps ‘latest'
-as the only tag. However, you can add any logic to this if you prefer:
+Remember to change how the image is tagged and where it's cached. The following example is a
+functional equivalent for the aforementioned logic against the GitHub Container Registry:
 
 {% raw %}
 ```yaml
-  tags: ghcr.io/<username>/simplewhale:latest
+  tags: ghcr.io/${{ secrets.GITHUB_USERNAME }}/simplewhale:latest
+  cache-from: type=registry,ref=ghcr.io/${{ secrets.GITHUB_USERNAME }}/simplewhale:buildcache
+  cache-to: type=registry,ref=ghcr.io/${{ secrets.GITHUB_USERNAME }}/simplewhale:buildcache,mode=max
```

Comment:
Don't think this is the right place to introduce cache behavior with GitHub Actions in a getting started guide. Also we should use `gha` cache instead.

Reply:
If you read the earlier documentation in this document, the caching behavior is already introduced. No instructions were provided when switching to the Github Container Registry.